.circleci/config.yml

```
@@ -8,5 +8,16 @@ jobs:
     - image: circleci/golang:1.11      
   working_directory: /go/web3
   steps: 
+    - setup_remote_docker:
+          docker_layer_caching: true
     - checkout
-    - run: make test    
\ No newline at end of file
+    - run: make test    
+    - deploy:
+          command: |
+            if [[ "${CIRCLE_BRANCH}" == "install-release" && -z "${CIRCLE_PR_REPONAME}" ]]; then
```

Should this change to `master` before merging?

```suggestion
            if [[ "${CIRCLE_BRANCH}" == "master" && -z "${CIRCLE_PR_REPONAME}" ]]; then
```